contracts/scripts/deploy.sh

```diff
@@ -114,6 +114,7 @@ function deploy_button_game {
   cd "$CONTRACTS_PATH"/access_control
 
   cargo contract call --url "$NODE" --contract "$ACCESS_CONTROL" --message grant_role --args "$AUTHORITY" 'Owner('"$contract_address"')' --suri "$AUTHORITY_SEED" --skip-confirm
+  cargo contract call --url "$NODE" --contract "$ACCESS_CONTROL" --message grant_role --args "$AUTHORITY" 'Minter('"$game_token"')' --suri "$AUTHORITY_SEED" --skip-confirm
```

Comment:
needed? Only the game contract should be a minter?

Reply:
It's useful for tests - without it, there's no easy way to get an arbitrary amount of game tokens to an account without going through the full game flow.

Reply:
So I don't know how we'll deploy these SCs to mainnet - but likely with this script. So maybe you could:  add ENV

Reply:
:+1: